pkg/serviceproxy/service_proxy.go

```diff
@@ -174,3 +222,111 @@ func (s *serviceProxy) validate() error {
 	}
 	return nil
 }
+
+func (s *serviceProxy) hubUserAuthenticatedAndInfo(token string) (bool, *authenticationv1.UserInfo, error) {
+	if userInfo, found := s.hubTokenCache.Get(token); found {
```

Comment:
I am not quite sure whether we really need the cache for now. Do we have any data on how this will improve? I would rather not using these type of cache for now.

Reply:
Agree, it's just a intuitive sense of me, no data back it. I will remove it.